service/matching/user_data_manager_test.go

```diff
@@ -788,7 +788,7 @@ func TestUserData_Propagation(t *testing.T) {
 		managers[i].config.GetUserDataLongPollTimeout = dynamicconfig.GetDurationPropertyFn(100 * time.Millisecond)
 		managers[i].config.GetUserDataMinWaitTime = 10 * time.Millisecond
 		managers[i].config.GetUserDataReturnBudget = 10 * time.Millisecond
-		managers[i].config.GetUserDataRetryPolicy = backoff.NewExponentialRetryPolicy(100 * time.Millisecond).WithMaximumInterval(1 * time.Second)
+		managers[i].config.GetUserDataRetryPolicy = backoff.NewExponentialRetryPolicy(100 * time.Millisecond).WithMaximumInterval(1 * time.Second).WithExpirationInterval(backoff.NoInterval)
```

Comment:
why is this `WithMaximumInterval(1 * time.Second)` instead of `WithMaximumInterval(1 * time.Minute)` like the config.go change?

Reply:
look a few lines up?
```
// override timeouts to run much faster
```

Reply:
ah sorry that didnt show up in my github diff until I clicked to show more lines. understood!